sidebars.js

```diff
@@ -36,8 +36,8 @@ const sidebars = {
           collapsed: false,
           items: [
             'getting-started/create-first-cluster/intro',
-            'getting-started/create-first-cluster/using_kubectl',
             'getting-started/create-first-cluster/using_fleet',
+            'getting-started/create-first-cluster/using_kubectl',
```

Comment:
I changed the order here according to the order in https://docs.rancher-turtles.com/docs/getting-started/create-first-cluster/intro so that it is easier to follow the guides